### PR TITLE
Chain view css updt

### PR DESCRIPTION
**Pull request by bubafistah**
# Description

Add styling for CHAIN view middle row, so that all cards would have the same height on all screen sizes.

Before
Screenshot from 2021-11-16 00-46-34
After
Screenshot from 2021-11-16 00-55-37

### CODE REVIEW

File: src/app/plugins/lthn/chain/blockchain.component.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 :host {
 
 }
+
+.vertical_fill {
```
Comment (Snider) [inline src/app/plugins/lthn/chain/blockchain.component.scss:5]: maybe `.vertical_fill { .mat-card { box-sizing: border-box;
	height: 100%; } }`
	then you can apply it to the row flex container? ish.

**Review by Snider — COMMENTED** [review]
This could maybe live in the root style.scss? (added a comment on the CSS, not sure if it'll get tagged in this)

**Comment by Snider** [comment]
Tyvm, eq-col's was a tweak I ran out of patience for at that point on that day :D

File: src/app/plugins/lthn/chain/blockchain.component.scss (hunk 1 above)
Comment (bubafistah) [reply src/app/plugins/lthn/chain/blockchain.component.scss:5]: Sure, that would be cleaner to just add it to container. My suggestion would be
```scss
.vertically-filled-mat-card-container {
	mat-card {
		box-sizing: border-box;
		height: 100%;
	}
}
```
or with some additional fanciness
```scss
@mixin vertical-fill {
	box-sizing: border-box;
	height: 100%;
}

.vertically-filled-mat-card-container {
	mat-card {
		@include vertical-fill;
	}
}
```
and
```html
<div class="vertically-filled-mat-card-container" fxLayout="row">
```

**Comment by bubafistah** [comment]
> This could maybe live in the root style.scss? (added a comment on the CSS, not sure if it'll get tagged in this)

Imo things should live only in module while it is not used elsewhere. When it becomes useful elsewhere, then move it up.
But you're the boss ;)

**Review by Snider — APPROVED** [review]
sweet, yea I'll use this a lot,  on lethean.hashvault.pro we use it on most row's, but its angularJS.

Trying to work up a mini display framework for plugin's which is my motivation to bubble up to framework :)

**Pull request merged**